app/models/price_group.rb

```diff
@@ -117,6 +118,15 @@ def setup_skip_review_price_policies
 
   private
 
+  def is_hidden_changed?
+    self.will_save_change_to_attribute?(:is_hidden)
```

Comment:
nit: `self` is unnecessary

Also, do we need to wrap `will_save_change_to_attribute?`?